contrib/node/src/python/pants/contrib/node/tasks/javascript_style.py

```diff
@@ -89,18 +93,32 @@ def _install_javascriptstyle(self, javascriptstyle_dir):
 
   def _run_javascriptstyle(self, target, javascriptstyle_bin_path, files, fix=False):
     args = [javascriptstyle_bin_path]
-    if fix:
+    if fix is True:
```

Comment:
any particular reason to specifically test fix is True?

Reply:
No, but I accidentally passed in a value other than a boolean while testing "lint" and it ran "fix mode". It was mainly a programming mistake, and according to pep 8, it is actually recommended not to do "if var is True". So will back out this change.